### ISSUE TITLE
Provide clarity on error resolution in Conflicting dependencies section

### ISSUE THREAD

**Issue by shrielenee** [magento/devdocs#4322]
# General issue

## Description:

This is an issue reported on the page via Hotjar feedback mechanism regarding the [Resolve component dependency readiness check issues page](https://devdocs.magento.com/guides/v2.1/comp-mgr/trouble/cman/component-depend.html#trouble-depend-conflict)

> I have deployed 'Magento certified by bitnami' on ec2 instances (t2 small) of AWS. Without touching any sort of code Anywhere I tried to install a module a couple of times though it showed the 'Conflicfting dependencies error'. I tried number of solutions like increasing the php memory in both php.ini files, even in .htaccess & user.ini but of no use. I tried with a command 'composer install --no-dev' and composer update --no-dev', this also failed. I removed the require-dev from the composrer.json though it worked but in stage 3 I get an error like 'rollback failed' and 'Updation already running' .

> "Edit composer.json to make any of these changes and try the readiness check again." Thats all what we get to know about editing composer.json. now, the question is what to edit in this?

## Possible solutions:

Improve clarity, in the [Conflicting dependencies section](https://devdocs.magento.com/guides/v2.1/comp-mgr/trouble/cman/component-depend.html#trouble-depend-conflict), on what to edit within the composer.json file to remedy the issue.

**Comment by tushars26** [comment]
I resized the the AWS ec2 Instance from t2.small to t2.medium and the readiness check been passed. 
Now, its a proven fact since we have invested days to sort it out, that Magento Community must consider, If you want to run Magento Stack do not go below 2GB RAM be it a Shared/Maganed hosting plan, VPC or anything ! 

Thanks to Address the Issue. Further, I request you to Add this Information in the Documentation too to ease the upcoming installations by people at learn.
